twitter/twitterapp/views.py

```diff
@@ -0,0 +1,18 @@
+from django.shortcuts import render
+from twitterapp.models import Tweet
+from django.http import HttpResponse
+
+# Create your views here.
+def all_tweets(request):
```

Comment:
Add a docstring for the view for documentation.

It is also useful to write the URL that this function will serve for future documentation.
```suggestion
def all_tweets(request):
    """Get all the tweets in the system.
    
    Since this returns everything from the database, this URL should be
    private or removed in future as it will cause very high load on the system.
    
    /twitter
    """
```